### DISCUSSION TITLE
Get in touch with ollama/ollama to become 'official' caller framework

### DISCUSSION THREAD

**Discussion by AgentSchmecker** [amithkoujalgi/ollama4j#27] [Ideas]
Hi to all,

I really like the way this repo is evolving and would love to see many more people to get to know what you are doing @amithkoujalgi. 
As far as I see, this repo is **the** central entrypoint for java developers when it comes to get a standardized way to communicate with the Ollama server side.

Hence it would be very handy to spread the word that this framework is existent and living. And the best way, at leas in my opinion, would be to get in contact with the maintainers of ollama/ollama itself, as they already mentioned two libraries

Libraries
[ollama-python](https://github.com/ollama/ollama-python)
[ollama-js](https://github.com/ollama/ollama-js)

for the languages TS and Python. Thus this repo would get the recognition of many ollama users that have some java background or are using java as base for their respective applications.

@amithkoujalgi What do you think?

Cheers!

**Comment by AgentSchmecker** [comment]
@amithkoujalgi what's your opinion on this topic? I've seen that there is a discord channel for ollama, maybe we could Post your GitHub repo there.

Greetings!

**Comment by amithkoujalgi** [comment]
Hi @AgentSchmecker! Thanks for bringing up this topic. I have seen that Ollama has added official libraries for Python and Javascript and these libraries are maintained in a GitHub organisation named Ollama, whereas libraries such as Ollama4j, OllamaSharp (for C#), etc., are being developed and maintained by community members such as ourselves in separate repositories. 
I believe that Ollama dev team has categorically added Python and JS libs as the official ones since they're maintaining it. 

However, I have got a [PR ](https://github.com/ollama/ollama/pull/1044) merged in the Ollama repository where I added Ollama4j to the list of community developed libraries.

![Screenshot (1)](https://github.com/amithkoujalgi/ollama4j/assets/1876165/6a45c885-e947-4db2-83d6-465968d2d8ae)

**Reply by AgentSchmecker** [reply]
Hi @amithkoujalgi, I've not taken notice of differenciation of "self-maintained" libraries and "community-driven" libraries. In this case your approach is perfect. Thanks a lot.

Other than that, I'd give a short reference on the discord channel as well to make people aware of this project, if you are okay with this :)

Cheers!

**Reply by amithkoujalgi** [reply]
Thanks a bunch for the initiative @AgentSchmecker! I appreciate your effort! I haven't used Discord in a while. 😅 So please go ahead.
